### ISSUE TITLE
Warnings and errors when running mainExampleRing.m

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/PESTO#57]
```
ans = 

  Figure (4: plotParameterUncertainty) with properties:

      Number: 4
        Name: 'plotParameterUncertainty'
       Color: [0.9400 0.9400 0.9400]
    Position: [1000 918 560 420]
       Units: 'pixels'

  Show all properties

Warning: No valid values for sigma found! Not plotting approximation. 
> In plotParameterUncertainty (line 613)
  In plotParameterSamples (line 43) 
Warning: No valid values for sigma found! Not plotting approximation. 
> In plotParameterUncertainty (line 613)
  In plotParameterSamples (line 43) 
Warning: No valid values for sigma found! Not plotting approximation. 
> In plotParameterUncertainty (line 613)
  In plotParameterSamples (line 43) 
Warning: No valid values for sigma found! Not plotting approximation. 
> In plotParameterUncertainty (line 613)
  In plotParameterSamples (line 43) 

ans = 

  Figure (5: plotParameterUncertainty) with properties:

      Number: 5
        Name: 'plotParameterUncertainty'
       Color: [0.9400 0.9400 0.9400]
    Position: [1000 918 560 420]
       Units: 'pixels'

  Show all properties

Reference to non-existent field 'MS'.

Error in plotConfidenceIntervals (line 156)
                    plot([pStruct.MS.par(iP,1), pStruct.MS.par(iP,1)], [j-0.4, j+0.4], 'k-', 'linewidth', 2);

Error in getParameterConfidenceIntervals (line 121)
        plotConfidenceIntervals(parameters, alpha, [], options);
```

**Comment by dweindl** [comment]
Minor: "define_Gauss_LLH.m" reference in the text does not exist

**Comment by BBallnus** [comment]
fixed. 

The problems were caused by the lines

par = rmfield(par, 'P');
par = rmfield(par, 'MS');

which were recently added. Gonna upload the fixed version in a couple of minutes.

**Comment by BBallnus** [comment]
fixed.

The problems were caused by the lines

par = rmfield(par, 'P');
par = rmfield(par, 'MS');

which were recently added. Gonna upload the fixed version in a couple of minutes.

**Comment by FFroehlich** [comment]
to better keep track of this: make a commit with "fixes #57" in the commit title. As soon as that commit reaches the master branch the issue will be automatically closed and the commit referenced here.

**Comment by FFroehlich** [comment]
Check https://github.com/ICB-DCM/AMICI/issues/19 as an example what this would look like.
